oeps/processes/oep-0062-proc-working-groups.rst

```diff
@@ -0,0 +1,191 @@
+----------------
+
+Evey working group must have a specific name and statement of pupose. The purpose can be brief: a sentance or a paragraph is sufficient. The name and purpose of can be amended at any time via consensus of working group members, although major changes should be announced to the community.
+
+Following this guideline will help community members understand the role of each working group in developing the Open edX platform. It will help them evaluate whether they would like to become involved in any given group.
+
+Chairperson
+-----------
+
+A working group must select by consensus a chairperson (or, a set of co-chairpeople) who should serve as a reliable point-of-contact for the group. The chairperson should work to ensure that the group follows the best practices specified in this document. Although chairpersons often take leadership in the group's activities and facilitate its meetings, this is not required. Chairpersons cannot be selected without their own consent.
+
+When a chairperson wishes to retire from their role, they should make a brief announcement detailing:
+
+* when they plan on stepping down,
+* any information that will help their successor transition into the role, and
+* optionally: any person or people who they recommend as their successor.
+
+The working group should aim to fill the role before the departing chairperson steps down. If they cannot, the group is encouraged to reach out the `Core Contributors`_ group for assisstance.
+
+Openness and Inclusivity
+------------------------
+
+As drivers of an open source project, Open edX working groups must make their activities as accessible to the public as reasonably possible. To this end, working groups should:
+
+* Welcome the participation of all community members who make good-faith, non-disruptive efforts to contribute to the group's purpose.
+* Collaborate using publicly-visible communication channels and/or task trackers.
+* Generate and review work artifacts (code, documents, images, decisions, etc.) in publicly-visible spaces.
+* Publicize any and all meeting times in UTC, with an open invite to Open edX community members.
+* Maintain an up-to-date listing of group metadata, including:
+
+  * **name**,
+  * **purpose**,
+  * **chairperson**,
+  * link(s) to **communication channels** and/or **task trackers**.
+  * link(s) to any spaces containing **work artifacts**, and
+  * link(s) to any additional documentation on group **structure and processes**.
+
+Working groups may exempt themselves from certain guidelines when they would compromise their ability to do work related to:
+
+* undisclosed security issues, or
+* unreleased marketing materials.
+
+Such groups should otherwise follow the openness & inclusivity guidelines to their best effort.
+
+Finally, as with any activity in the Open edX community, behavior in working groups that conflicts with the `Open edX Code of Conduct`_ should be swiftly by the means detailed in that policy.
```

Comment:
Swiftly... dealt with? 🤔  Sounds sinister. 😂

Reply:
whoops, swiftly _addressed_.

Not meant to be sinister 😁 Just trying to be clear: although groups need to be inclusive, they don't need to tolerate jerkish behavior. In fact, to be most inclusive, they should address jerkish behavior as soon as possible. Happy to reword.

Reply:
Reworded this a bit.